pkgs/servers/sql/postgresql/ext/pointcloud/package.nix

```diff
@@ -0,0 +1,121 @@
+{
+  autoconf,
+  automake,
+  cunit,
+  fetchFromGitHub,
+  installShellFiles,
+  lib,
+  libtool,
+  libxml2,
+  perl,
+  postgresql,
+  postgresqlBuildExtension,
+  postgresqlTestExtension,
+  postgresqlTestHook,
+  sphinx,
+  which,
+  zlib,
+}:
+postgresqlBuildExtension (finalAttrs: {
+  pname = "pointcloud";
+  version = "1.2.5";
+
+  outputs = [
+    "out"
+    "man"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "pgpointcloud";
+    repo = "pointcloud";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-uFbScxq21kt0jOjjyfMeO3i+bG2/kWS/Rrt3ZpOqEns=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    installShellFiles
+    # for xml2-config
+    libxml2
```

Comment:
`configure: error: could not find libxml2` during cross-compilation.
```suggestion
```
Fix with:
```nix
  configureFlags = [
    (lib.withFeatureAs true "xml2config" (lib.getExe' (lib.getDev libxml2) "xml2-config"))
  ];
```

Reply:
Try `nix build -L .#pkgsCross.aarch64-multiplatform.postgresqlPackages.pointcloud`.